### ISSUE TITLE
Can't demo course created in Swirlify

### ISSUE THREAD

**Issue by scaevola01** [swirldev/swirl#653]
I'm trying to create an intro course in Swirlify to support a class on using R for Historical Linguistics at UCLA. However, when I try to demo the course that I've created, I get the following error message:

> >demo_lesson()
> 
> | Course installed successfully!
> 
> Error in lesson(df, lesson_name = meta$Lesson, course_name = meta$Course,  : 
>   Argument 'df' must be a data frame!
> 
> | Leaving swirl now. Type swirl() to resume.

As a result, I can't demo what I've created, and I can't currently justify to my partners investing more time in creating Swirl courses (which I really want to do). Does anyone understand the error, and can someone help me resolve it? I'm using Swirlify 0.5.1, Swirl 2.4.3, RStudio 1.0.153, R 3.4.2, and Mac OS 10.13.1.

**Comment by seankross** [comment]
What do you get as a result of `test_course()` or `test_lesson()`?

**Comment by scaevola01** [comment]
Here's my results:

> > test_course()
> It seems this course does not contian a LICENSE.txt file.
> You can easily add a license with add_license().
> 
> Error in test_course() : 
>   It seems there's no MANIFEST file for this course.
> Please add one using add_to_manifest().
> > test_lesson()
> ##### Begin testing: Intro to R #####
> ##### End testing: Intro to R #####

**Comment by seankross** [comment]
It looks like you need to add a MANIFEST file with a listing for every lesson in your course. 

- You can see an example here: https://github.com/swirldev/swirl_courses/blob/master/R_Programming/MANIFEST. 
- Also see: http://swirlstats.com/swirlify/structure.html#organizing_lessons

**Comment by scaevola01** [comment]
Unfortunately, that doesn't solve the problem:

> > add_to_manifest()
> > demo_lesson()
> 
> | Course installed successfully!
> 
> Error in lesson(df, lesson_name = meta$Lesson, course_name = meta$Course,  : 
>   Argument 'df' must be a data frame!
> 
> | Leaving swirl now. Type swirl() to resume.
> 

I also added a License file, which did not solve the problem, either.

**Comment by seankross** [comment]
Is the course on GitHub?

**Comment by scaevola01** [comment]
No. I'm just trying to create a proof-of-concept on my local machine.

**Comment by seankross** [comment]
Can you zip up the course and email it to me?

**Comment by scaevola01** [comment]
Sure. What's the best email?

**Comment by seankross** [comment]
sean at seankross dot com
